### ISSUE TITLE
Failure in name resolution when downloading the model

### ISSUE THREAD

**Issue by Sachin-Wani** [zaandahl/mewc#2]
I followed all the instructions to train a model (specifically the EN-V2-S) for an Island Conservation project. 
It correctly detects my dataset and goes to next step of downloading the .h5 file but fails to do so.

<img width="666" alt="image" src="https://github.com/zaandahl/mewc/assets/25133376/782c6bf7-7d4d-4e93-87e4-d92f9298e62f">

<img width="715" alt="image" src="https://github.com/zaandahl/mewc/assets/25133376/4ee2e189-65de-4768-b6b1-d7437fb4ffef">

And I tried using the .h5 file downloaded and saved in data folder and passed in .env as variable value. No matter if I use full path, relative path, ./path, I get an OS error for that. Maybe I am doing something wrong here. But thought I will let you know. I am looking forward to making this work. Let me know if you need more information on this. 

Thanks!

**Comment by zaandahl** [comment]
Hi @Sachin-Wani , the error looks like a problem downloading from https://storage.googleapis.com (possibly related to your DNS resolution). 

I was able to successfully download the efficientnetv2-s_notop.h5 model using the Docker container as a test. I'd recommend simply trying to run the command again, or potentially restarting your workstation to clear out any cache. You can also see if you can download directly in your browser as the download link that Python uses is just an https URL. 

Please let me know if this helps and if you are able to run the training step with the container.

Regards,
Zach

`Downloading data from https://storage.googleapis.com/tensorflow/keras-applications/efficientnet_v2/efficientnetv2-s_notop.h5
82420632/82420632 [==============================] - 4s 0us/step`

**Comment by zaandahl** [comment]
Another thought is to check network settings (proxy, firewall, etc...). Docker has settings for manually configuring proxy and network, but by default these should be turned off. As long as everything is default, that shouldn't be a problem. You can see if it is an external network problem by testing the command connected to a different network (e.g. if you are trying at home, try the command on your work or university network or vice versa).

**Comment by Sachin-Wani** [comment]
Hey, so I am not sure exactly what the issue is, but using my personal laptop, I was able to download the model and run the training steps. Using the workstation (which I have to connect to through a proprietary VPN) did not download the model. I will look into fixing that on my end. Thanks for the comments. 

Just another thing to note: The classifier models use a Focal Loss from tensorflow-addons (tfa) which may be deprecated in the future, we can switch that to polyl or other module (like keras-cv) to prevent issues popping up in the future. 

Many thanks for this repo!
